lib/matobj.gi

```diff
@@ -1346,14 +1273,8 @@ InstallMethod( AdditiveInverseSameMutability,
     end );
 
 InstallMethod( ZeroMutable,
-    [ IsMatrixOrMatrixObj ],
-    function( M )
-    if IsPlistRep( M ) then
-      TryNextMethod();
-    else
-      return Matrix( ZeroMutable( Unpack( M ) ), M );
-    fi;
-    end );
+    [ IsMatrixObj ],
+    M -> Matrix( ZeroMutable( Unpack( M ) ), M ) );
```

Comment:
Should we perhaps instead delegate to ZeroMatrix?
```suggestion
    M -> ZeroMatrix( NrRows( M ), NrCols( M ), M ) );
```

Reply:
Yes, thanks. (As a rule of thumb, avoiding `Unpack` is an improvement.)